.travis.yml

```diff
@@ -1,5 +1,14 @@
 language: java
 
+env:
+  - DOCKER_COMPOSE_VERSION=1.18.0
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
```

Comment:
Why remove the local docker-compose?

Reply:
I get an error in the early version (see https://travis-ci.org/bakdata/conquery/builds/339475952)